content/hardware/Other-components/CANBUS.md

```diff
@@ -24,8 +24,16 @@ CAN connector pinout:
 
 ![image](../../img/can-connector-pinout.png)
 
-Massey Ferguson - steer bus location:
+Case / New Holland
+Activate the necessary in the [HH menu](../../files/NH_H3_Config.pdf)
+There's also a CH3 which MUST be NO else you'll have errors.
+Maybe you need to [calibrate the WAS as well](../../files/NH_Steering_Cal.pdf)
```

Comment:
This will all be rendered on a single line. Is that what you wanted? Maybe you could use a header (`#` or `##`) to structure the different brands?